try {
    // Dangerous code
} catch(ExceptionType e) {
    // e -- exception variable, contains exception info
}
